.config/dotnet-tools.json

```diff
@@ -0,0 +1,12 @@
+{
+    "version": 1,
+    "isRoot": true,
+    "tools": {
+      "dotnet-format": {
+        "version": "4.1.131201",
+        "commands": [
+          "dotnet-format"
```

Comment:
Is this file using 2 or 4 space indentation? Easy to fix though :) either way should be fine.